static/js/invite.js

```diff
@@ -105,6 +105,15 @@ exports.initialize = function () {
                               .show();
                 invitee_emails_group.addClass('warning');
 
+                if (arr.activation_msg !== '') {
+                    invite_status.append($('<p>').text(arr.activation_msg));
+                    if (arr.activation_link !== '') {
+                        invite_status.find('p:last-child').append($('<a>').attr('href', '#organization/deactivated-users-admin')
+                            .text(i18n.t('organization settings')).css({color: 'inherit', 'text-decoration': 'underline'}))
```

Comment:
Why is `organization settings` lower-case here?  

In general, we're not super into doing this sort of complex hacking of the HTML in JS; it'd be better to have a `hidden` element in the template with the right CSS, and just have JS show that element.